examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -91,7 +91,7 @@
 cv = LeaveOneLabelOut(session)
 
 # Compute the prediction accuracy for the different folds (i.e. session)
-cv_scores = cross_val_score(anova_svc, X, conditions)
+cv_scores = cross_val_score(anova_svc, X, conditions, cv = cv)
```

Comment:
In Python, there are coding style rules (known as PEP8) that keyword arguments (here "cv", should not have spacing before or after the "=" sign. Would you mind removing those space, so that we keep the style consistent in the examples?

Cheers